std/typecons.d

```diff
@@ -5189,6 +5189,8 @@ mixin template Proxy(alias a)
                 return a.opCmp(b);
             else static if (is(typeof(b.opCmp(a))))
                 return -b.opCmp(a);
+            else static if (isFloatingPoint!ValueType || isFloatingPoint!B)
+                return a < b ? -1 : a > b ? +1 : a == b ? 0 : float.nan;
```

Comment:
Oh, I see. Post inlining, some of the work done by the simpler implementation gets eliminated. I guess we're in good shape. Could you please operate the stylistic change for the integral comparison, and let's finish. Thanks!